django-stubs/contrib/auth/backends.pyi

```diff
@@ -7,12 +7,12 @@ from django.db.models.base import Model
 from django.http.request import HttpRequest
 from typing_extensions import TypeAlias
 
-UserModel = get_user_model()
```

Comment:
I get stubtest errors if I leave this as it. Not entirely sure why. See the CI failures here: https://github.com/typeddjango/django-stubs/actions/runs/6267951533/job/17022010362

Reply:
If it exist at runtime we can start out by exposing it, regardless of public API or not I'd say

Reply:
It exists at runtime, but the types seem to be different, so subtest complains. Is there a way to ignore things from stubtest? Is that what the allowlist is for?

Reply:
Yes precisely. Update the allowlist, that's what it's for

Reply:
But before we do, can you inspect what the runtime type is?

Reply:
This is what subtest says at least:

```
error: django.contrib.auth.views.UserModel.username variable differs from runtime type django.db.models.query_utils.DeferredAttribute
Stub: in file /home/runner/work/django-stubs/django-stubs/django-stubs/contrib/auth/views.pyi:76
django.db.models.fields.CharField[Union[builtins.str, builtins.int, django.db.models.expressions.Combinable], builtins.str]
Runtime:
<django.db.models.query_utils.DeferredAttribute object at 0x7f3b7d36c850>
```

Seems to be the same for all the attributes

Reply:
Aha, interesting, that's a descriptor: https://github.com/django/django/blob/4de31ec680df062e5964b630f1b881ead5004e15/django/db/models/query_utils.py#L178

Should be fine to type as "what it really is"